types/src/transaction/mod.rs

```diff
@@ -738,13 +736,88 @@ impl TransactionExecutableRef<'_> {
 }
 
 #[derive(Clone, Debug, Hash, Eq, PartialEq, Serialize, Deserialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+pub struct ScheduledTxnConfig {
+    pub allow_rescheduling: bool,
+    pub expiration_time: u64,
+    pub authorization_seqno: u64,
+}
+
+#[derive(Clone, Debug, Hash, Eq, PartialEq, Serialize, Deserialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+pub enum PermissionsType {
+    ScheduledTxnsPermissions,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+pub struct PermissionsTbl {
```

Comment:
cc: @igor-aptos to take a look too, whether it's extensible to permission signer model